README.md

```diff
@@ -302,117 +296,6 @@ client = Client(key=NEXMO_API_KEY, secret=NEXMO_API_SECRET)
 verify = Verify(client)
 ```
 
-### Search for a Verification request
```

Comment:
Should this be pulled out? Seems like we're not actually removing the Verify Class so I'd say keep this documented as it's considered best practice and just tag the methods as deprecated

Reply:
Added that back in, the old diff made it look like we had an entire extra section for code snippets as this was signposted differently